lyricsbot/domains/songlyrics/config.py

```diff
@@ -0,0 +1,4 @@
+"""
+Format URL link for config.
```

Comment:
Config file should cover all cases like constants or private data. But I see in module doc-string, that this file only for one case.

Reply:
Done.